timemachine/cpp/src/context.hpp

```diff
@@ -72,6 +72,8 @@ class Context {
 
     double _get_temperature();
 
+    void _ensure_local_md_intialized();
```

Comment:
nit: should this be `private`?

Reply:
Yes, `setup_local_md` is the preferred way to do this. It makes it explicit and allows modification to the behavior.

`_ensure_local_md_initialized` is just for internal usage, though should it should call `setup_local_md`

Reply:
`_ensure_local_md_initialized` now calls `setup_local_md` in [addad67](https://github.com/proteneer/timemachine/pull/1014/commits/addad672327d50ab8b9454af0681fbd80a033d57)